src/features/content-insights/types.ts

```diff
@@ -23,3 +23,11 @@ export type MetricsData = {
 export type GraphDatum = TimeRange & MetricsData;
 
 export type GraphData = Array<GraphDatum>;
+
+export type ResponseError = Error & {
```

Comment:
Does it need to include the `Error` type?

Reply:
The error type has fields `message` and `stack`, I found both those fields to exist when taking a look at a 500 error I made in the accessstats api